### ISSUE TITLE
deleteCloudContainerLocalOnly:YES Deletes All Devices

### ISSUE THREAD

**Issue by kdbdallas** [lhunath/UbiquityStoreManager#29]
I ran `[self.ubiquityStoreManager deleteCloudContainerLocalOnly:YES];` on my Mac app expecting to only delete the contents of that device, however instead it deleted the contents on all devices, just as if I had specified `NO` instead of `YES`

**Comment by lhunath** [comment]
Can you provide USM logs of what happens when you do this?

**Comment by kdbdallas** [comment]
Of course. I had a work Christmas party which included an overnight stay at a fancy hotel, so I had to run after finding this, and now trying to play catch up work being a grown up :( Should be able to in a couple hours (that means I am going to loose my data again!!! NOOOO!!!)

:)

**Comment by lhunath** [comment]
well, if you do a `setCloudDisabledAndOverwriteLocalWithCloudIfConfirmed:` to force a migration from cloud -> local, then do a `[self.ubiquityStoreManager deleteCloudContainerLocalOnly:YES];` you can always do a `setCloudEnabledAndOverwriteCloudWithLocalIfConfirmed:` any time later to migrate local back to cloud and re-populate if your cloud store did get emptied :-)

**Comment by lhunath** [comment]
... or just `migrateCloudToLocal` and `migrateLocalToCloud`.
